system/jlib/jlog.hpp

```diff
@@ -845,9 +845,10 @@ inline LogMsgCategory MCexception(IException * e, LogMsgClass cls = MSGCLS_error
     return LogMsgCategory((e)->errorAudience(),cls, mapClassToDefaultDetailLevel(cls));
 }
 
-#define MCerror MCuserError
-#define MCwarning MCuserWarning
-#define MCprogress MCuserProgress
+#define MCstats LogCategory_OperatorWarning
```

Comment:
Not really an equivalent category, and we shouldn't be  adding more #defines - should delete them if anyything because they don't match the pattern of (audience, category)